### DISCUSSION TITLE
API client testing

### DISCUSSION THREAD

**Discussion by mpounsett** [peeringdb/peeringdb#1443] [Q&A] [answered]
Is there a version of the site with non-production data, where API client code can be tested?

**Comment by mcmanuss8** [comment, marked as answer]
Beta.peeringdb.com-Steve(from iPhone - expect typos)On Sep 14, 2023, at 6:37 PM, Matthew Pounsett ***@***.***> wrote:﻿
Is there a version of the site with non-production data, where API client code can be tested?
